extra/onnx.py

```diff
@@ -75,12 +75,26 @@ def run_onnx(inputs={}, debug=False):
       elif n.op_type == "MatMul": ret = inp[0].matmul(inp[1])
       # one liners
       elif n.op_type == "Elu": ret = inp[0].elu(alpha=opt['alpha'])
-      elif n.op_type == "Clip": ret = inp[0].clip(*(inp[1:] if len(inp) > 1 else (opt['min'], opt['max'])))
+      elif n.op_type == "Clip": ret = inp[0].clip(*(inp[1:] if len(inp) > 1 else (opt.get('min', -3.4e38), opt.get('max', 3.4e38))))
```

Comment:
float("-inf") and float("inf") instead?

Reply:
Just saw this, that works too. can change in another PR